src/app/@ansyn/map-facade/services/map-facade.service.ts

```diff
@@ -99,26 +101,28 @@ export class MapFacadeService {
 		}
 	}
 
-	exportMapsToPng() {
-		this.store.select(selectMapsIds).pipe(
-			take(1),
-			switchMap((mapsIds: string[]) => {
-				const _maps = [];
-				mapsIds.forEach((mapId) => {
-					const provider = this.imageryCommunicatorService.provide(mapId);
-					const exportData: ICanvasExportData = provider.ActiveMap.getExportData();
-					_maps.push(exportData);
-				});
-				return mapsToPng(_maps, this.layout)
-			}),
-			tap(blob => saveAs(blob, 'map.jpeg')),
-			tap(() => {
-				this.store.dispatch(new ExportMapsToPngActionSuccess());
-			}),
-			catchError((err) => {
-				this.store.dispatch(new ExportMapsToPngActionFailed(err));
-				return EMPTY;
-			})
-		).subscribe();
+	async exportMapsToPng(element: Element | string = null) {
+		this.store.dispatch(new SetMinimalistViewModeAction(true));
+		await setTimeout(async () => {
```

Comment:
i think you should remove this method from here
add action of start export, end export, and in the map facade register to it and do your thing there